### ISSUE TITLE
Action.execute().subscribe() receiving ActionError.notEnabled even when enabled

### ISSUE THREAD

**Issue by asutula** [RxSwiftCommunity/Action#75]
I could be misunderstanding this, but my thinking is that subscription closures passed into `Action.execute().subscribe()` should receive values, errors, etc from that particular invocation of `execute()`. I noticed I was never getting called back in the `onCompleted` closure, and found that I always receive a `ActionError.notEnabled`. This happens even when I create my Action with an explicit `enabledIf: Observable.just(true)`.

**Comment by fpillet** [comment]
Can you give a go to the HEAD of the repo? we think we fixed this issue, a second confirmation would be great.

**Comment by fpillet** [comment]
BTW errors are being passed to the observable returned for each execution. Do you have a case where this doesn't work?

**Comment by ishkawa** [comment]
I confirmed we've already fixed this issue in our HEAD.
This patch will get released [today](https://github.com/RxSwiftCommunity/Action/pull/73#issuecomment-271165989) 🙇 

### Test code

```swift
import UIKit
import RxSwift
import Action

class ViewController: UIViewController {
    var action: CocoaAction!

    override func viewDidLoad() {
        super.viewDidLoad()

        action = CocoaAction {
            return Observable
                .just()
                .delay(1.0, scheduler: MainScheduler.instance)
        }

        action.execute().debug().subscribe()
    }
}
```

### 2.1.0 output

```
2017-01-09 15:43:13.820: TestViewController.swift:17 (viewDidLoad()) -> subscribed
2017-01-09 15:43:13.821: TestViewController.swift:17 (viewDidLoad()) -> Event error(notEnabled)
Received unhandled error: /Users/ishkawa/dev/src/github.com/RxSwiftCommunity/Action/Demo/TestViewController.swift:17:viewDidLoad() -> notEnabled
2017-01-09 15:43:13.821: TestViewController.swift:17 (viewDidLoad()) -> isDisposed
```

### HEAD output

```
2017-01-09 15:44:36.798: TestViewController.swift:17 (viewDidLoad()) -> subscribed
2017-01-09 15:44:37.811: TestViewController.swift:17 (viewDidLoad()) -> Event next(())
2017-01-09 15:44:37.812: TestViewController.swift:17 (viewDidLoad()) -> Event completed
2017-01-09 15:44:37.812: TestViewController.swift:17 (viewDidLoad()) -> isDisposed
```

**Comment by fpillet** [comment]
Ohhh you have a reproducible test? Fantastic, we'll be able to add it to the tests then! 👍

**Comment by asutula** [comment]
I just switched to HEAD and it works! Thank you all.

**Comment by ashfurrow** [comment]
Great! Thanks a lot for testing this! Going to close the issue, this should be released as 2.2.0 shortly.
